<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget_main">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>1280</width>
      <height>600</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QLabel" name="label_homepage">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>531</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Home Page. empty for now- we could put the login here, extra information, or remove it if we don't need it</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>160</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>have buttons now just for testing permissions</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>230</y>
        <width>160</width>
        <height>98</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>manager</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>employee</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>admin</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_POSpage">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>190</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>page for our POS system</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QStackedWidget" name="stackedWidget_sales">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>600</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_sales_1">
       <widget class="QLabel" name="label_dailysales">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>240</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Page for saily sales reports</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_sales_2">
       <widget class="QLabel" name="label_membersales">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>220</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>page for sales sorted by member</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_sales_3">
       <widget class="QLabel" name="label_inventorysales">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>260</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>page for sales sorted by item</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>120</y>
        <width>160</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_sales">
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_sales_member">
         <property name="text">
          <string>Sort by Member</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_sales_item">
         <property name="text">
          <string>Sort by Item</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_sales_daily">
         <property name="text">
          <string>Daily Sales</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_membership">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>230</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>page for membership info- (member list, rebate suggestions)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QStackedWidget" name="stackedWidget_admin">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>600</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_admin_1">
       <widget class="QLabel" name="label_admin_member">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>270</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>page for adding/deleting customers</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_admin_2">
       <widget class="QLabel" name="label_admin_inventory">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>240</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>page for adding/deleting inventory</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>110</y>
        <width>160</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_admin">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_admin_member">
         <property name="text">
          <string>Manage Members</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_admin_inventory">
         <property name="text">
          <string>Manage Inventory</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>491</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_main">
     <item row="0" column="6">
      <widget class="QPushButton" name="pushButton_admin">
       <property name="text">
        <string>Administrator</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_POS">
       <property name="text">
        <string>POS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_home">
       <property name="text">
        <string>Home </string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_sales">
       <property name="text">
        <string>Sales Reports</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_members">
       <property name="text">
        <string>Membership Info</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
